exercicios/para-casa/entregas/laura-leamari/resolucao.js

```diff
@@ -0,0 +1,53 @@
+class ContaBancaria {
+  constructor(titular, saldo, limite) {
+    this.titular = titular;
+    this.saldo = saldo;
+    this.limite = limite;
+  }
+
+  consultarSaldo() {
+    return this.saldo;
+  }
+
+  desativarConta(ativa) {
```

Comment:
acredito que uma outra forma também de fazer seria incluir uma função que conseguisse ativar e desativar o limite. no meu código, eu inclui dentro do constructor o atributo limiteAtivo e implementei a seguinte função. 

 alternarLimite() {
    if (this.limiteAtivo) {
      this.limiteAtivo = false;
      this.limiteConta = 0;
      return `${this.nomeTitular}, o limite da sua conta foi desativado.`;
    } else {
      this.limiteAtivo = true;
      return `${this.nomeTitular}, o limite da sua conta foi ativado.`;
    }
  }